packages/integration-tests/src/package_manager_sanity_checks.test.ts

```diff
@@ -25,6 +25,7 @@ import {
   runPackageManager,
   runWithPackageManager,
 } from './process-controller/process_controller.js';
+import { amplifyAtTag } from './constants.js';
```

Comment:
it would be great to somehow make it parametrizable so we could use it in snapshot releases.

Reply:
Created: https://github.com/aws-amplify/amplify-backend/issues/1349